papers/copper-mountain-2021/lfa_pmultigrid_high_order_fem.tex

```diff
@@ -0,0 +1,973 @@
+While it is possible \cite{davydov2019matrix} to use high-order finite element methods with matrix-free implementation of h-multigrid preconditioning on complex problems, p-multigrid can be a more natural fit for high-order methods on unstructured meshes.
+
+In this paper, we develop LFA of p-multigrid with arbitrary second-order PDEs using high-order finite element discretizations and investigate Jacobi and Chebyshev smoothing for two-grid schemes and introduce LFAToolkit.jl \cite{thompson2021toolkit}, a new Julia package for LFA of high-order finite element methods.
+There are multiple software packages that offer LFA of h-multigrid methods, such as \cite{rittich2018extending,kahl2020automated}.
+However, to the best of our knowledge, no packages support LFA of p-multigrid methods, especially with arbitrary PDEs.
+Although \cite{van2011discrete} discussed LFA of p-multigrid for discontinuous Galerkin method, this formulation cannot be extended to the continuous Galerkin method.
+Our LFA formulation for p-multigrid with high-order finite element discretizations for the continuous Galerkin method can be generalized to reproduce previous work on h-multigrid for high-order finite elements \cite{he2020two} and can be extended to LFA for h-multigrid of finite difference discretizations that can be represented with finite element discretizations.
+
+We investigate the performance of Jacobi and Chebyshev semi-iterative method smoothers for p-multigrid with aggressive coarsening for the scalar Laplacian in one and two dimensions and validate our LFA formulation with the scalar Laplacian in three dimensions.
+Traditional estimates of the optimal Jacobi smoothing parameter are ill-suited to p-multigrid for high-order elements, and the performance of p-multigrid with these two polynomial smoothers degrades as we coarsen more aggressively.
+Finally, to demonstrate capability of our LFA of p-multigrid, the performance of three dimensional linear elasticity with a Chebyshev smoother is presented.
+
+This paper is organized as follows.
+In \cref{sec:notation} we outline the notation for LFA used in this paper.
+In \cref{sec:lfa} we develop the notation for LFA of second-order PDEs with arbitrary order bases, dimension, and number of components used in LFAToolkit.jl, and we use this notation to develop LFA of p-multigrid as well as Jacobi and Chebyshev smoothers.
+\Cref{sec:results} contains numerical results investigating the performance of Jacobi and Chebyshev smoothing for p-multigrid on the one, two, and three dimensional scalar Laplacian and three dimensional linear elasticity.
+Finally, \cref{sec:conclusion} contains concluding remarks.
+
+% -----------------------------------------------------------------------------
+\subsection{Reproducibility}\label{sec:reproducibility}
+% -----------------------------------------------------------------------------
+
+%\yh{remove this subsection to Numerical section?}
```

Comment:
Personally, I don't feel like enough papers make it straightforward enough to reproduce their results, which is why I put this section in a prominent location. I can move it if we think it is important, but I would rather make a statement about the importance of reproducibility and how easy our work is to replicate by making this statement very obvious to the reader.